ml-algorithms/src/main/java/org/opensearch/ml/engine/Executable.java

```diff
@@ -15,6 +17,5 @@ public interface Executable {
      * @param input input data
      * @return execution result
      */
-    Output execute(Input input);
-
+    Output execute(Input input) throws Exception;
```

Comment:
Generally we should avoid throwing `Exception` it's too general. Should throw specific exception, like `IOException` etc.
For this general interface API , I think it's hard to tell which concrete exception we should throw.

We can catch exception and throw runtime exception inside `execute`.